marlowe-dashboard-client/static/css/main.css

```diff
@@ -1,4 +1,4 @@
-@import url("https://fonts.googleapis.com/css2?family=Barlow:wght@400;600;700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Barlow:ital,wght@0,400;0,600;0,700;1,400&display=swap");
```

Comment:
what do these changes do?

Reply:
These import the proper italic version of the font. I just noticed that it was missing while I was doing this PR (because italic text shows up in the hints). The browser's automatic rendering of the italic version doesn't look as good.